.github/workflows/workflow.yml

```diff
@@ -26,6 +27,7 @@ jobs:
     runs-on: ubuntu-latest
     env:
       JASYPT_ENCRYPTOR_PASSWORD: ${{secrets.JASYPT_ENCRYPTOR_PASSWORD}}
+      PROBLEM_ACCESS_PASSWORD: ${{secrets.PROBLEM_ACCESS_PASSWORD}}
```

Comment:
Can you document this in the README? For the running locally step

Reply:
Good idea - done! 👍